### ISSUE TITLE
Text cursor focus is not pulled to alt text modal when opened

### ISSUE THREAD

**Issue by Juicysteak117** [bluesky-social/social-app#2224]
**Describe the bug**

When clicking on an image to add alt text in the post composer, the text cursor doesn't get moved to the alt text modal. This is mildly annoying as it means an extra click is required before typing can begin, but the primary bug concern is that it means pasting (ctrl+V) still goes to the drafted post text when highlighted instead of the alt text modal. In other words, drafted words are being overwritten, but ONLY when highlighted prior. Typing does nothing always and pasting otherwise does nothing.

**To Reproduce**

1. Draft a post with some text.
2. Attach an image.
3. Fill clipboard with text or image.
4. Highlight drafted text.
5. Click on image to open alt text modal (on Firefox, the ALT button explicitly must be clicked, not anywhere on the image)
6. Paste using ctrl+V
7. Observe text being replaced by clipboard or deleted entirely in the case of an image.

**Expected behavior**

When opening the alt text modal, text cursor focus should be pulled to the "Add alt text" composer and anything pasted should go there.

**Details**

 - Platform: Windows 10 desktop browser agnostic
 - Platform version:
 - App version:

**Comment by haileyok** [comment]
Yea this one is definitely fixed now 👍 thank you!
